export const getOrder = async (req, res, next) => {
    try {
        
    } catch (error) {
        next(error);
    }
};
export const addOrder = async (req, res, next) => {
    try {

    } catch (error) {
        next(error);
    }
};
export const updateOrder = async (req, res, next) => {
    try {

    } catch (error) {
        next(error);
    }
};
export const deleteOrder = async (req, res, next) => {
    try {

    } catch (error) {
        next(error);
    }
};
